XBeeLibrary.Xamarin/release_notes.txt

```diff
@@ -2,10 +2,10 @@
                                XBeeLibrary.Xamarin
```

Comment:
The `XBeeLibraray.Core` module has also a release notes file, I think we should update it too.